internal/compression_structures_test.go

```diff
@@ -155,3 +155,19 @@ func TestCompressAndEncryptError(t *testing.T) {
 		t.Errorf("compress: CompressingPipeWriter expected CompressAndEncryptError but got %v", re)
 	}
 }
+
+func TestCompressAndEncryptWithNoCompression(t *testing.T) {
+	for _, tt := range tests {
```

Comment:
What means tt? Two twats?

Reply:
No idea - I've copypasted it from other test. I'll change it to something more understandable)